### PR TITLE
:white_check_mark: (CI/CD): Added deploy workflow #1022

### PR DESCRIPTION
**Pull request by brandon-c-k**
close

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+name: Deploy
+
+on:
+  push:
+    branches: [ release ]
```
Comment (priyo5) [inline .github/workflows/deploy.yml:6]: Add a wildcard to release to allow for multiple iterations of the branch see full.yml for reference

**Review by priyo5 — CHANGES_REQUESTED** [review]
See comment made in file

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (brandon-c-k) [reply .github/workflows/deploy.yml]: @priyo5 noted

**Comment by arneschreuder** [comment]
Closing, no more PRs to develop

**Pull request closed without merging**